### ISSUE TITLE
New scalings for light captured and total leaf area

### ISSUE THREAD

**Issue by qdread** [qdread/forestscalingworkflow#1]
Per recent discussion with @jgradym I am adding two new scalings.

- Light captured (this is the total light flux, `light_received` multiplied by `fraction_light_captured`, the correction factor taking into account that not all light passing through the crown volume can be absorbed by the tree)
- Leaf area (this is crown area multiplied by the estimated leaf area index for a given crown depth)

What needs to be done?

- [x] Update scripts for the part of the pipeline before model fitting (creating the data)
- [x] Update model fitting commands
- [x]  Update scripts for the part of the pipeline after model fitting (extracting model output and creating summary tables and data for plotting) -- scripts 7, 10, 12, and 14
- [x] Running the appropriate scripts before model fitting (scripts 4 and 6 at least)
- [x] Running models
- [x] Rerunning any models that did not converge the first time
- [x] Running post-model fitting scripts (scripts 7, 10, 11, 12, 13, and 14)
- [x] Check over plotting data script to ensure new data is incorporated. Make new plot scripts if needed.

**Comment by qdread** [comment]
Initial progress made in 30f4c55

**Comment by qdread** [comment]
Code finished editing, but not tested or run yet, in 286e09b (not yet including any plotting code)

**Comment by qdread** [comment]
All models are finished running and have converged except for the "unclassified" trees for the leaf area scaling, and functional group 3 (slow) for the light captured scaling. We can probably move ahead without the unclassified trees model because it's fairly unimportant but I will try to make sure that FG3 converges.

**Comment by jgradym** [comment]
Sweet! Will check out

**Comment by qdread** [comment]
Model for FG3 converged. So now all models are run. The folder on google drive `ForestLight/data/clean_summary_tables` now contains the parameter estimates, Bayesian R-squared, and information criteria CSVs for the light captured and leaf area scalings. Next step is to create plotting data, working on that now.

**Comment by qdread** [comment]
New plotting script was created, I committed a new script to the forestlight repo at <https://github.com/qdread/forestlight/blob/master/code/plotting/additional_plots_3may2021.R> with three new plots, indicating the location where I modified code in clean_plotting_code.R. I did not want to directly replace anything in that script yet, because of all the specific formatting and axis limits and things like that, in case you still want to use the old plots. I believe this takes care of everything.
